app/tasks/email_notifications.rb

```diff
@@ -23,7 +21,7 @@ def contest_starting(time_text)
     users = notif.users
 
     Ajat.info "contest_starting|id:#{@contest.id}|time:#{time_text}"
-    send_emails(text: text, subject: subject, users: users)
+    send_emails(text: text, subject: subject, users: users, event: 'contest_starting')
```

Comment:
Can you use a symbol instead of string for these events?

i.e. `:contest_starting` instead of `'contest_starting'`.